### PR TITLE
[ENH] Add information to FiffInfo

### LINKED ISSUE #543: Add support to read line frequency from fiff file

### PR DESCRIPTION
**Pull request by RDoerfel**
This PR adds following information to FiffInfo:

- linefreq
- proj_id
- xplotter_layout
- experimenter   
- description     
- utc_offset      
- gantry_angle

It closes #543

### CODE REVIEW

File: examples/ex_hpiFit/main.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ int main(int argc, char *argv[])
     parser.setApplicationDescription("hpiFit Example");
     parser.addHelpOption();
     qInfo() << "Please download the mne-cpp-test-data folder from Github (mne-tools) into mne-cpp/bin.";
-    QCommandLineOption inputOption("fileIn", "The input file <in>.", "in", QCoreApplication::applicationDirPath() + "/MNE-sample-data/chpi/raw/data_with_movement_chpi_raw.fif");
+    QCommandLineOption inputOption("fileIn", "The input file <in>.", "in", QCoreApplication::applicationDirPath() + "/mne-cpp-test-data/MEG/sample/test_hpiFit_raw.fif");
```
Comment (LorenzE) [inline examples/ex_hpiFit/main.cpp:99]: Was this intended?

File: libraries/fiff/fiff_info.cpp [hunk 2]
```diff
@@ -59,6 +59,7 @@ using namespace Eigen;
 FiffInfo::FiffInfo()
 : FiffInfoBase()//nchan(-1)
 , sfreq(-1.0)
+, linefreq(-1.0)
```
Comment (LorenzE) [inline libraries/fiff/fiff_info.cpp:62]: add a default value for `gantry_angle `

File: libraries/fiff/fiff_stream.cpp [hunk 3]
```diff
@@ -2019,9 +2076,7 @@ FiffStream::SPtr FiffStream::start_writing_raw(QIODevice &p_IODevice, const Fiff
 
         t_pStream2 = FiffStream::SPtr();
     }
-    //
-    // Unused parameters from original file (could eventually put these in info struct)
-    //
+
     QList<fiff_int_t> values;
```
Comment (LorenzE) [inline libraries/fiff/fiff_stream.cpp:2080]: This code block (line 2080 to 2103) can be removed if you write the tags `FIFF_EXPERIMENTER, FIFF_DESCRIPTION, FIFF_PROJ_ID, FIFF_PROJ_NAME, FIFF_LINE_FREQ, FIFF_XPLOTTER_LAYOUT` to stream later in this function below the [General comment](https://github.com/mne-tools/mne-cpp/blob/04e1c62ceb276521168c981f96853ea163cf7d1c/libraries/fiff/fiff_stream.cpp#L2105).

File: examples/ex_hpiFit/main.cpp (hunk 1 above)
Comment (RDoerfel) [reply examples/ex_hpiFit/main.cpp:99]: Yes, I didn't set the example back to it's public state in the last PR. I usually use this example to get my results for the hpiFit. So I always have to change file pahts etc.

**Pull request merged**